source/Jobbr.Server/Scheduling/Planer/PlanResult.cs

```diff
@@ -2,14 +2,14 @@
 
 namespace Jobbr.Server.Scheduling.Planer
 {
-    internal struct PlanResult
+    public struct PlanResult
```

Comment:
Why the visibility change?

Reply:
It is used as a return type in public methods.

Reply:
And did that visibility change or how could it be internal before?

Reply:
JobRunPlaners [sic] used to have internal methods, but the methods are now public due to them having interfaces.